frontend/src/pages/transactions/details/Details.tsx

```diff
@@ -276,7 +277,12 @@ const Details: FunctionComponent = () => {
             GRCP Status:
           </Label>
           <Value variant="large" className={classes.inlineValue}>
-            {FlowUtils.getGrcpStatusName(transaction.status?.statusCode)}
+            <div className={classes.flex}>
+              <StatusIcon status={transaction.status?.statusCode} />
+              <div className={classes.text}>
+                {FlowUtils.getGrcpStatusName(transaction.status?.statusCode)}
+              </div>
+            </div>
```

Comment:
Do you think it would make sense to move that markup to a seperate reusable component (e.g. `GrcpStatusBadge`)? Because I think we could easily reuse it somewhere.

Reply:
Maybe yes, but we use only the status icon at /transactions and then we use icon and text at /transactions/details to display the transactions status. Maybe if the badge has an custom option whether to display text?

Reply:
Yep, I think it would make sense to then extract this to a seperate component and add some kind of flag (e.g. `showStatusName`) to indicate whether to show the "detailed" badge that includes the status text.

I was was thinking of showing some kind of "error message tooltip" in the grcp status component, in case the transaction status contains an populated "error message" field. So by reusing this component in every place we show grcp status we could very easily implement that.